src/main/java/com/google/sps/servlets/EventLoaderServlet.java

```diff
@@ -0,0 +1,52 @@
+// Copyright 2019 Google LLC
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     https://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+
+package com.google.sps.servlets;
+
+import java.io.IOException;
+
+import com.google.api.services.calendar.model.Event;
+import com.google.sps.api.CalendarInterface;
+
+import java.io.PrintWriter;
+import java.util.List;
+
+import javax.servlet.annotation.WebServlet;
+import javax.servlet.http.HttpServlet;
+import javax.servlet.http.HttpServletRequest;
+import javax.servlet.http.HttpServletResponse;
+
+import java.io.IOException;
+
+@WebServlet("/load_events")
+public class EventLoaderServlet extends HttpServlet {
+
+    private final CalendarInterface calendarInterface = new CalendarInterface();
+
+    public EventLoaderServlet() throws IOException {
+    }
+
+    @Override
+    protected void doGet(HttpServletRequest request, HttpServletResponse response) throws IOException {
+        response.setContentType("text/html");
+        response.setCharacterEncoding("UTF-8");
+        PrintWriter writer = response.getWriter();
+        List<Event> events = calendarInterface.loadPrimaryCalendarEventsOfTomorrow();
+        for (Event event : events) {
+            writer.println(event.getSummary());
+            writer.println("\n");
```

Comment:
this prints two new lines but they won't really show up in the browser since it's html which ignores whitespace. Maybe a `<p>` or `<br>` tag are better fits if you want to render proper HTML.